var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -127,6 +127,8 @@ class Llvm(CMakePackage, CudaPackage):
         "or as a project (with the compiler in use)",
     )
 
+    conflicts("libcxx=runtime", when="@14 %apple-clang@15:")
+
```

Comment:
Please, try replacing this conflict with [this](https://github.com/skosukhin/spack/commit/c1607ce6276290255be142d4d6d7818e50d0c7b5).

Reply:
Seems to work fine, thanks !